dbt_subprojects/daily_spellbook/models/_sector/dex/pools/optimism/uniswap/_sources.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+ 
+sources:
+  - name: uniswap_v3_optimism
+    tables:
+      - name: UniswapV3Factory_evt_PoolCreated
```

Comment:
wondering if this spell is more useful as optimism pools?
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/models/_projects/uniswap/optimism/uniswap_optimism_pools.sql

OP team built this and it's used in uni v3 optimism dex trades model:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/models/trades/optimism/platforms/uniswap_v3_optimism_base_trades.sql#L18

Reply:
interesting that they read a different factory decoded table -- maybe the contract was decoded twice, with different names?